src/main/kotlin/eu/europa/ec/eudi/openid4vci/internal/formats/IssuanceRequestJsonMapper.kt

```diff
@@ -89,7 +106,7 @@ private fun IssuanceResponseEncryptionSpec.transferObject(): CredentialResponseE
 
 @Serializable
 @OptIn(ExperimentalSerializationApi::class)
-@JsonClassDiscriminator("format")
+@JsonClassDiscriminator("_type_")
 internal sealed interface CredentialIssuanceRequestTO {
```

Comment:
Consider removing the sealed hierarchy for TOs.

We just need one TO representing the Credential Request and a function
that  given a domain credential request (batch or single) returens the JSON representation